static/app/views/releases/detail/overview/chart/releaseChartControls.tsx

```diff
@@ -151,13 +151,15 @@ const ReleaseChartControls = ({
   return (
     <ChartControls>
       <InlineContainer>
-        <SectionHeading key="total-label">{getSummaryHeading()}</SectionHeading>
+        <SectionHeading key="total-label">
+          {getSummaryHeading()}
+          <QuestionTooltip
+            position="top"
+            size="sm"
+            title={t('This value includes only the current release.')}
+          />
+        </SectionHeading>
         <SectionValue key="total-value">{summary}</SectionValue>
-        <QuestionTooltip
-          position="top"
-          size="sm"
-          title={t('This value includes only the current release.')}
-        />
```

Comment:
is this related to the breakpoint changes?

Reply:
Yeah it's cause I made changes to InlineContainer and we already had centering styles for a tooltip in SectionHeading. Will post screenshot